docs/develop/develop-add-ins-visual-studio.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Develop Office Add-ins with Visual Studio
+description: How to develop Office Add-ins with Visual Studio
+ms.date: 12/28/2019
+localization_priority: Priority
+---
+
+# Develop Office Add-ins with Visual Studio
+
+This article describes how to use Visual Studio to develop an Office Add-in. If you've already created your add-in, you can skip ahead to the [Develop the add-in using Visual Studio](#develop-the-add-in-using-visual-studio) section.
+
+> [!NOTE]
+> As an alternative to using Visual Studio, you may choose to use the Yeoman generator for Office Add-ins and VS Code to create an Office Add-in. For more information about this choice, see [Creating an Office Add-in](../overview/office-add-ins-fundamentals.md#creating-an-office-add-in).
+
+## Create the add-in project using Visual Studio
+
+Visual Studio can be used to create Office Add-ins for Excel, Outlook, Word, and PowerPoint. An Office Add-in project gets created as part of a Visual Studio solution and uses HTML, CSS, and JavaScript. To create an Office Add-in with Visual Studio, follow instructions in the quick start that corresponds to the add-in you'd like to create:
+
+- [Excel quick start](../quickstarts/excel-quickstart-jquery.md?tabs=visualstudio)
+- [Outlook quick start](/outlook/add-ins/quick-start?context=office/dev/add-ins/context&tabs=visualstudio)
+- [Word quick start](../quickstarts/word-quickstart.md?tabs=visualstudio)
+- [PowerPoint quick start](../quickstarts/powerpoint-quickstart.md?tabs=visualstudio)
+
+Visual Studio does not support creating Office Add-ins for OneNote or Project. To create Office Add-ins for either of these hosts, you'll need to use the Yeoman generator for Office Add-ins, as described in the [OneNote quick start](../quickstarts/onenote-quickstart.md) or the [Project quick start](../quickstarts/project-quickstart.md).
+
+## Develop the add-in using Visual Studio
+
+Visual Studio creates a basic add-in with limited functionality. You can customize the add-in by editing the [manifest](add-in-manifests.md), HTML, JavaScript, and CSS files in Visual Studio. For a high-level description of the project structure and files in the add-in project that Visual Studio creates, see the Visual Studio guidance within the quick start that you completed to create your add-in. 
+
+> [!TIP]
+> Because an Office Add-in is a web application, you'll need at least basic web development skills to customize your add-in. If you're new to JavaScript, we recommend reviewing the [Mozilla JavaScript tutorial](https://developer.mozilla.org/docs/Web/JavaScript/Guide/Introduction).
```

Comment:
Is it worth combining this tip with the one about Yeoman above? Both approaches would require JS - just trying to figure out a way to get the "you need to know JS" closer to the top of the page.

Reply:
Hmmmmm...IMO, this info doesn't really fit with the contents of the note at the top of the page -- it addresses an entirely separate point/topic than the note about Yeoman does. Let's leave this tip where it is for now, and we can reevaluate if we receive feedback that indicates it'd be more helpful to customers earlier on in the article.